pilot/utils/files.py

```diff
@@ -23,7 +31,6 @@ def setup_workspace(args):
         return args['workspace']
 
     root = args.get('root') or get_parent_folder('pilot')
-    create_directory(root, 'workspace')
```

Comment:
why is this removed?

Reply:
Because `os.makedirs` does this anyway

>def makedirs(name, mode=0o777, exist_ok=False):
>    """
>    Super-mkdir; create a leaf directory and all intermediate ones.  Works like
>    mkdir, except that any intermediate path segment (not just the rightmost)
>    will be created if it does not exist.